docs/links/open-collective.py

```diff
@@ -0,0 +1,7 @@
+from . import link
+
+link_name = "open-collective" 
+link_text = "Become a member on Open Collective" 
```

Comment:
I think this is maybe a bit too specific and it prevents re-use when just referring to Open Collective, perhaps just use 'Open Collective' as the link text and write the rest of the words in the text around it? WDYT?